src/main/java/io/supertokens/inmemorydb/queries/GeneralQueries.java

```diff
@@ -371,6 +374,13 @@ public static long getUsersCount(Start start, RECIPE_ID[] includeRecipeIds) thro
         return finalResult;
     }
 
+    public static DeleteUserResult deleteUser(Start start, @Nonnull String userId)
+            throws StorageQueryException, StorageTransactionLogicException {
+        DeleteUserQuery query = new DeleteUserQuery(start);
```

Comment:
You can make the `execute` function a status function in `DeleteUserQuery`. Cause there is no value in creating an object out of this since you simply want to "namespace" that query.